front/lib/models/assistant/actions/dust_app_run.ts

```diff
@@ -1,13 +1,80 @@
 import type { DustAppParameters } from "@dust-tt/types";
 import type {
   CreationOptional,
+  ForeignKey,
   InferAttributes,
   InferCreationAttributes,
 } from "sequelize";
 import { DataTypes, Model } from "sequelize";
 
+import { AgentConfiguration } from "@app/lib/models/assistant/agent";
```

Comment:
let's use the default model import path

Reply:
🙏

Reply:
Actually... doing this creates a circular dependency issue. I'd be in favor of deleting this index file, as it just tends to create such problems.